### ISSUE TITLE
Accessible IasCard

### ISSUE THREAD

**Issue by Sl1nd** [inovex/elements#1004]
### Feature Request

* Selected State of IasCard should be accessible

### Description of the proposed solution

Propably the selected state should be reflected in [aria-checked](https://www.w3.org/TR/wai-aria-1.0/states_and_properties#aria-checked) state
